users_groups_setup.py

```diff
@@ -66,20 +64,16 @@ def mylar3(self):
         os.system(
             '/bin/bash -c "sudo useradd mylar -u 13005'
             ' ; sudo mkdir -pv ' + self.root_dir + '/data/{media,usenet,torrents}/comics -m 775'
-            ' ; sudo chown -R mylar:mediacenter ' + self.root_dir + '/data/{media,usenet,torrents}/comics'
-            ' ; sudo chown $(id -u):mediacenter ' + self.root_dir + '/data'
-            ' ; sudo chown $(id -u):mediacenter ' + self.root_dir + '/data/{media,usenet,torrents}"'
+            ' ; sudo chown -R mylar:mediacenter ' + self.root_dir + '/data/{media,usenet,torrents}/comics"'
         )
         self.create_config_dir('mylar')
         os.system('sudo usermod -a -G mediacenter mylar')
 
     def audiobookshelf(self):
         os.system(
-            '/bin/bash -c "sudo useradd audiobookshelf -u 13009'
-            ' ; sudo mkdir -pv ' + self.root_dir + '/data/{media,usenet,torrents}/audiobooks -m 775'
-            ' ; sudo chown -R audiobookshelf:mediacenter ' + self.root_dir + '/data/{media,usenet,torrents}/audiobooks'
-            ' ; sudo chown $(id -u):mediacenter ' + self.root_dir + '/data'
-            ' ; sudo chown $(id -u):mediacenter ' + self.root_dir + '/data/{media,usenet,torrents}"'
+            '/bin/bash -c "sudo useradd audiobookshelf -u 13014'
+            ' ; sudo mkdir -pv ' + self.root_dir + '/data/media/{audiobooks,podcasts,audiobookshelf-metadata} -m 775'
+            ' ; sudo chown -R audiobookshelf:mediacenter ' + self.root_dir + '/data/media/{audiobooks,podcasts,audiobookshelf-metadata}"'
```

Comment:
Do we also want to put stuff here for torrents/usenet? I'm not sure whether audiobookshelf has a feature to move stuff when torrents/usenet downloads (?) complete like Sonarr/Radarr does, but we do make those for movies and tv shows.